.github/workflows/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy
+azure-storage-blob
+azure-identity
```

Comment:
Is this file supposed to be inside the folder .github/worklows ?

Reply:
I think usually you want the requirements.txt to be at the root of your project.

The command usually used is `pip freeze > requirements.txt`

![image](https://github.com/ai-cfia/nachet-metadata-format/assets/96267006/1e4d4b93-b247-4e15-aa63-063d2f0a8923)

Reply:
fixed